### ISSUE TITLE
100€ bounty: Loading-bar should not move messages

### ISSUE THREAD

**Issue by Erstus** [1Hyena/cryptograffiti#30]
After loading-bar finishes, its height will animate to zero. Firefox plays the animation through and resets the height to its initial value. This results a yanking movement. Reason of this animation would be to make room for content, but If the loading-bar is positioned absolutely, it would not create an extra gap between messages at all. I suggest doing this and removing the animation completely. That would be faster and cleaner.

**Comment by Erstus** [comment]
I just realized the title part of this issue has been solved in some point. I'm not sure when or by whom. But the animation ends smoothly. It could be that some Firefox update changed it, or it was a side-effect of some commit. Still, the animation has questionable purpose, so instead of closing, i think this issue should be renamed to: "Loading-bar should not move messages". Removing the need for this animation would not just be a technical optimization, but it would also result in a cleaner visual experience. One argument for this is that if anyone tries to read some message while the other is loading, the movement would disrupt their attention.
